Database/input-data/pmount/031e922a53f27d2c3f48656fef07c655c7418103de14fbac4d21b083d1ece50c.json

```diff
@@ -0,0 +1,43 @@
+{
+    "Conventions":"CF-1.8",
+    "activity_id":"input4MIPs",
+    "comment":"The NASA NOAA LASP (NNL) solar variability models were formerly known as the Naval Research Laboratory (NRL) solar variability models. NNL V1 models will become the operational NOAA/NCEI Solar Irradiance Climate Data Record (CDR) V3 in August 2024. The SSI and F10.7 data are taken from V03. Sub-annual variability has been added for the period before 1874; TSI in this file is the integral over SSI from source data between 0 and 100,000nm",
+    "comment_post_publication":null,
+    "contact":"bernd@iaa.es",
+    "creation_date":"2024-10-14T09:25:37Z",
+    "data_node":null,
+    "dataset_category":"solar",
+    "datetime_end":"2023-12-31T12:00:00Z",
+    "datetime_start":"1850-01-01T12:00:00Z",
+    "esgf_dataset_master_id":"input4MIPs.CMIP6Plus.CMIP.SOLARIS-HEPPA.SOLARIS-HEPPA-CMIP-4-4.atmos.day.multiple.gn.v20241018",
+    "filepath":"/p/user_pub/work/input4MIPs/CMIP6Plus/CMIP/SOLARIS-HEPPA/SOLARIS-HEPPA-CMIP-4-4/atmos/day/multiple/gn/v20241018/multiple_input4MIPs_solar_CMIP_SOLARIS-HEPPA-CMIP-4-4_gn_18500101-20231231.nc",
+    "frequency":"day",
+    "further_info_url":"http://solarisheppa.geomar.de/cmip7",
+    "grid":null,
+    "grid_label":"gn",
+    "institution":"APARC SOLARIS-HEPPA",
+    "institution_id":"SOLARIS-HEPPA",
+    "latest":null,
+    "license":"Solar forcing data produced by SOLARIS-HEPPA is licensed under a Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/). The data producers and data providers make no warranty, either expressed or implied, including but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.",
+    "license_id":null,
+    "mip_era":"CMIP6Plus",
+    "nominal_resolution":"10000 km",
+    "product":"derived",
+    "publication_status":"in_publishing_queue",
```

Comment:
This is published now - so this needs a toggle

Reply:
I think the previous comment got lost. This is just a placeholder. These files are generated from the stuff from pmount, which doesn't know anything about the status on ESGF, hence this value will always be wrong. If you want to look at the JSON (which I don't recommend, because it's painful to parse), you have to look at this file: Database/input4MIPs_db_file_entries.json